localization/english/culture/cultures_l_english.yml

```diff
@@ -269,37 +269,37 @@
  #Indo Aryan
  indo_aryan_group:0 "Indo-Aryan"
  indo_aryan_group_collective_noun:0 "Indo-Aryans"
- bengali:0 "Bangala"
- bengali_collective_noun:0 "Bangala"
- oriya:0 "Oria"
- oriya_collective_noun:1 "Oria"
+ bengali:0 "Vaṅgālīya"
+ bengali_collective_noun:0 "Vaṅgāla"
+ oriya:0 "Oḍrīya"
+ oriya_collective_noun:1 "Oḍra"
  assamese:0 "Asamiya"
  assamese_collective_noun:0 "Asamiya"
- hindustani:0 "Hindustani"
- hindustani_collective_noun:0 "Hindustani"
- gujarati:0 "Gujarati"
- gujarati_collective_noun:1 "Gujarats"
- punjabi:0 "Punjabi"
- punjabi_collective_noun:1 "Punjabs"
- rajput:0 "Rajputra"
- rajput_collective_noun:1 "Rajputra"
- sindhi:0 "Sindhi"
- sindhi_collective_noun:1 "Sindhs"
- marathi:0 "Marathi"
- marathi_collective_noun:1 "Marathis"
- sinhala:0 "Sinhala"
- sinhala_collective_noun:1 "Sinhalas"
- nepali:0 "Nepali"
- nepali_collective_noun:1 "Nepalese"
+ hindustani:0 "Madhyadeśīya"
+ hindustani_collective_noun:0 "Madhyadeśīya"
+ gujarati:0 "Gurjara"
+ gujarati_collective_noun:1 "Gurjara"
+ punjabi:0 "Pāñcanada"
+ punjabi_collective_noun:1 "Pāñcanada"
+ rajput:0 "Rājapūtra"
+ rajput_collective_noun:1 "Rājapūtra"
+ sindhi:0 "Sindhī"
+ sindhi_collective_noun:1 "Sindhī"
+ marathi:0 "Mar‍hāṭī"
+ marathi_collective_noun:1 "Marhāṭe"
```

Comment:
I have no idea what changing 'i' from 'e' achieved here. I'll bring up these issues in the sheet as well